[0.0.2a36 regression] `.column_valued()` without a `name` is no longer accepted

**Describe the bug**
Commit 20436f71520e26e1f6fd81836ee7e283d69db8df (#258) incorrectly removed the default for the `name` parameter of `FunctionElement.column_valued`, which is needed to accept calls without a `name`, i.e., `.column_valued()`.

**Expected behavior**
This call should be accepted.

**To Reproduce**

```py
from sqlalchemy import func

func.unnest([1, 2, 3]).column_valued()
```

**Error**

```
test.py:3: error: Missing positional argument "name" in call to "column_valued" of "FunctionElement"  [call-arg]
Found 1 error in 1 file (checked 1 source file)
```

**Versions.**
 - OS: Linux (NixOS 23.11)
 - Python: 3.11.5
 - SQLAlchemy: 1.4.50
 - mypy: 1.7.0
 - SQLAlchemy2-stubs: 0.0.2a36

**Additional context**
<!-- Add any other context about the problem here. -->

**Have a nice day!**

--- a/sqlalchemy-stubs/sql/functions.pyi
+++ b/sqlalchemy-stubs/sql/functions.pyi
@@ -59,7 +59,7 @@ class FunctionElement(  # type: ignore[misc]
     ) -> ScalarFunctionColumn[_TE]: ...
     def table_valued(self, *expr: Any, **kw: Any) -> TableValuedAlias: ...  # type: ignore[override]
     def column_valued(
-        self, name: Optional[Any], joins_implicitly: bool = False
+        self, name: Optional[Any] = ..., joins_implicitly: bool = False
     ) -> TableValuedColumn[Any]: ...
     @property
     def columns(self) -> ColumnCollection[ColumnElement[Any]]: ...  # type: ignore[override]

